README.md

```diff
@@ -1,5 +1,4 @@
-Drupal
-===
+<img alt="Drupal Logo" src="https://www.drupal.org/files/Wordmark_blue_RGB.png" height="60px">
```

Comment:
If we want extra clarification for text-only users, we could consider something like this:
```suggestion
<!-- Drupal Logo -->
<img alt="Drupal Logo" src="https://www.drupal.org/files/Wordmark_blue_RGB.png" height="60px">
<!-- End Drupal Logo -->
```

Reply:
@gabesullice And maybe we don't even need the `<!-- End Drupal Logo -->`?
```suggestion
<!-- Drupal Logo -->
<img alt="Drupal Logo" src="https://www.drupal.org/files/Wordmark_blue_RGB.png" height="60px">

```